include/revng/ABIAnalyses/Analyses.h

```diff
@@ -0,0 +1,61 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include "revng/ABIAnalyses/Generated/DeadRegisterArgumentsOfFunction.h"
+#include "revng/ABIAnalyses/Generated/DeadReturnValuesOfFunctionCall.h"
+#include "revng/ABIAnalyses/Generated/RegisterArgumentsOfFunctionCall.h"
+#include "revng/ABIAnalyses/Generated/UsedArgumentsOfFunction.h"
+#include "revng/ABIAnalyses/Generated/UsedReturnValuesOfFunction.h"
+#include "revng/ABIAnalyses/Generated/UsedReturnValuesOfFunctionCall.h"
```

Comment:
Are these generated in the source directory?

Reply:
not anymore

Reply:
now using cmake like `monotone-framework.py`